techdebt: [CDS-86988]: Troubleshoot Doc Updation (#4741)
harness/developer-hub: docs/continuous-delivery/gitops/gitops-ref/troubleshooting.md

- A different scope (account, organization, or project) in Harness.

- A different Argo CD project that is not mapped to a Harness project.
  

### Issue: "rpc error: code = Unknown desc = error testing repository connectivity: authorization failed"

> This error occured when attempted to connect a repo with installed agent and it's unable to connect repository in Github:

Reviewer: This error occurs while attempting to connect a repository:

> This error occured when attempted to connect a repo with installed agent and it's unable to connect repository in Github:

Reviewer 2: ```suggestion
This error occurs when an agent is unable to connect to a repo:
```

> - Make sure to monitor rate limitations on github account before attempting to connect repo.

Reviewer: GitHub

> - Make sure to monitor rate limitations on github account before attempting to connect repo.

Reviewer 2: ```suggestion
- Ensure to review Github account rate limitations before attempting to connect to a repo.
```

> - Manage 'Rate Limiting' in Git by 'Enable Git Rate Limiting' to True.

Reviewer: where is this setting in Git?

> - Manage 'Rate Limiting' in Git by 'Enable Git Rate Limiting' to True.

Reviewer 2: ```suggestion
- To manage rate limiting in GitHub, see [Enabling rate limits for Git](https://docs.github.com/en/enterprise-server@3.10/admin/configuration/configuring-user-applications-for-your-enterprise/configuring-rate-limits#enabling-rate-limits-for-git).
```
